src/Twig/PayPlugExtension.php

```diff
@@ -13,21 +14,33 @@ final class PayPlugExtension extends AbstractExtension
 {
     /** @var CanSaveCardCheckerInterface */
     private $canSaveCardChecker;
+    private PayPlugApiClientFactory $apiClientFactory;
 
-    public function __construct(CanSaveCardCheckerInterface $canSaveCardChecker)
-    {
+    public function __construct(
+        CanSaveCardCheckerInterface $canSaveCardChecker,
+        PayPlugApiClientFactory $apiClientFactory,
+    ) {
         $this->canSaveCardChecker = $canSaveCardChecker;
+        $this->apiClientFactory = $apiClientFactory;
     }
 
     public function getFunctions(): array
     {
         return [
             new TwigFunction('is_save_card_enabled', [$this, 'isSaveCardAllowed']),
+            new TwigFunction('is_payplug_live', [$this, 'isLive']),
```

Comment:
`is_[payplug_]test_mode_enabled` is preferred considering [payplug integrated api](https://docs.payplug.com/api/integratedref.html#constructor) constructor signature